packages/oui-navbar/src/menu/navbar-menu.html

```diff
@@ -31,7 +31,7 @@ <h3 class="oui-navbar-menu__title" ng-bind="::$ctrl.headerTitle"></h3>
                 'oui-navbar_desktop-only': !!menuLink.subLinks,
                 'oui-navbar-link_external': !!menuLink.isExternal
             }"
-            ng-click="$ctrl.navbarCtrl.toggleMenu()"
+            ng-click="menuLink.linkClicked(menuLink);$ctrl.navbarCtrl.toggleMenu()"
```

Comment:
`linkClicked` has been made for notifications (from what i see in the documentation). 
We already have a property for this: `click`. For now, it's only active on button. Take a look below :)

```
ng-click="$ctrl.closeMenuWithCallback($event, menuLink.click)"
```

This sould be done on `<oui-navbar-links>` too, by adding an `on-click` attribute :)